tools/rpkg/R/s3_register.R

```diff
@@ -0,0 +1,71 @@
+# From: https://github.com/r-lib/vctrs/blob/main/R/register-s3.R
```

Comment:
There's now a newer version of this file in the rlang repo.

Reply:
Thanks, I'll update it.